Grid Header Template fix and kb linking (#388)
telerik/blazor-docs: knowledge-base/grid-wrap-and-center-column-text.md

## Notes

> * If you only need to center the header text content, you can also try the approach demonstrated in this knowledge base article - [Center Grid Column Header content]({%slug grid-kb-center-column-header-content%}). It allows you to keep the preset `display: flex` of the cells and simply justify their content to the center.

Reviewer: ```suggestion
* If you only need to center the header text content, you can try the approach demonstrated in this knowledge base article - [Center Grid Column Header content]({%slug grid-kb-center-column-header-content%}). This will keep the preset `display: flex` style of the cells.
```
